src/app/(content-layout)/[groupId]/editgroup/_editgroup/DeleteGroupButton.tsx

```diff
@@ -6,16 +6,21 @@ import DangerModal from '@/components/danger-modal';
 import useModalContext from '@/components/common/modal/core/useModalContext';
 import { useRouter } from 'next/navigation';
 import { Toast } from '@/components/common/Toastify';
+import { useUser } from '@/contexts/UserContext';
 
 const DELETE_MODAL_ID = 'delete-group';
 
 export default function DeleteGroupButton({ groupId }: { groupId: number }) {
   const { openModal } = useModalContext();
+  const { fetchUser } = useUser();
   const router = useRouter();
 
   const handleDeleteGroup = async () => {
     await deleteGroup(groupId)
-      .then(() => router.push('/'))
+      .then(() => {
+        fetchUser();
+        router.push('/');
+      })
```

Comment:
`fetchUser`가 Promise를 리턴하기 때문에

```suggestion
await deleteGroup(groupId)
      .then(() => fetchUser())
      .then(() => {
        router.push('/');
      })
```

가 더 적절할 것 같습니다.

Reply:
코멘트 감사합니다!! 위에 코드는 제가 직접 작성한 부분은 아니지만, 제 파트와 연결되는 수정이 필요해서 함께 PR에 포함되었습니다 😅 말씀해주신 내용은 팀원분께도 전달드리고, 필요한 부분은 함께 반영하겠습니다.